docs/reports/communitybuilding/minutes_2025.md

```diff
@@ -1,5 +1,15 @@
 # Jupyter Community Building Working Group 2025 Meeting Minutes
 
+## 20 February 2025
+Attendees: Jason, Ana, Martha
+* Brainstorm ideas for adding people to the JCB committee. Would want to vet people coming on, so would have a form people can fill out.
+  * Appeal to UoC
+  * Create a blog post that includes a form to apply to the committee
+  * JupyterCon: submit a panel or breakout session on working groups and standing committees
+  * Governing board: perhaps people at those organizations would like to volunteer for Jupyter in this capacity
```

Comment:
Wording change suggestion: perhaps members of this board have people at their organizations that would like to join?